DOCUMENTATION.md

```diff
@@ -615,6 +616,87 @@ var page = PO.build({
 
 Note that if the plain object doesn't have attributes defined, the object is returned as is.
 
+### `.customHelper`
+
+Defines a way in which custom components can be created using
+information of their surrounding context.
+
+It expects a function with the following signature:
```

Comment:
If you look at the other attributes, we add a _attribute signature_ section after the short description.

Something like...

```markdown
Attribute signature

` ``js
PO.customHelper(userFunction);
` ``

Where the provided user function has the following signature

` ``js
function(selector, options) {
  // user __code__ goes here
  return Value;
}
` ``

...

```

Or something on the line